src/simulator/callback.h

```diff
@@ -5,36 +5,81 @@
 
 
 #include "common.h"
+#include "utils/memory.h"
+#include "utils/os.h"
 #include "containers/string.h"
 #include "simulator/network.h"
 
 
+/*****************************
+*  NETWORK DUMPER definitions
+*****************************/
+// TODO: Can't I use this directly in the neuron class???
+typedef struct DumperNeuronData {
```

Comment:
Remove TODO, the normal neuron needs to be as small as possible and it keeps only the epsc and ipsc

Reply:
done